### ISSUE TITLE
pdfsizeopt.main.PdfTokenParseError: unrecognized PDF signature '%PDF-1.3%\r\n1  0 '

### ISSUE THREAD

**Issue by rbrito** [pts/pdfsizeopt#85]
Hi, @pts.

I have been going through a lot of files lately and some of them are giving pdfsizeopt a hard time. Some are understandably non-compliant, but they are common enough that major PDF viewers display them correctly and without problems.

In particular, I have a very, very similar issue to https://github.com/pts/pdfsizeopt/issues/76, where I have some PDF files (creadted by a major retail shop here in Brazil) that have incorrect signatures...

Since you were so fast at fixing the regexp there, I didn't have the time to also add the problem to that problem report.

Anyway, this is what I get with the latest pdfsizeopt when running on one such file:

```
$ ~/Downloads/pdfsizeopt/pdfsizeopt --use-multivalent=no --do-optimize-images=no nfe.pdf 
info: This is pdfsizeopt rUNKNOWN size=393191.
info: prepending to PATH: /home/rbrito/Downloads/pdfsizeopt
info: loading PDF from: nfe.pdf
info: loaded PDF of 41890 bytes
Traceback (most recent call last):
  File "/home/rbrito/Downloads/pdfsizeopt/pdfsizeopt", line 41, in <module>
    sys.exit(main.main(sys.argv, script_dir=script_dir))
  File "/home/rbrito/Downloads/pdfsizeopt/lib/pdfsizeopt/main.py", line 9480, in main
    ).Load(file_name)
  File "/home/rbrito/Downloads/pdfsizeopt/lib/pdfsizeopt/main.py", line 4522, in Load
    raise PdfTokenParseError('unrecognized PDF signature %r' % data[: 16])
pdfsizeopt.main.PdfTokenParseError: unrecognized PDF signature '%PDF-1.3%\r\n1  0 '
```

The file in question contains some sensitive information and I can send it to you privately, if you wish. BTW, according to `pdfinfo` it was created by some Microsoft software stack:

```
$ pdfinfo /tmp/nfe.pdf 
Title:          rpts_dnfe_retrato_padrao_v3_1
Subject:        
Author:         
Creator:        Microsoft Reporting Services 9.0
Producer:       Microsoft Reporting Services PDF Rendering Extension 9.0
(...)
```

Thanks in advance for your help,

Rogério Brito.

**Comment by pts** [comment]
Thank you for reporting this! Fixed it in 43e7c05099ad8d9153623ad4d427a7299fc94b4c.

**Comment by rbrito** [comment]
Wow, that was fast! :-) Thanks a lot, @pts!
